full_snapshot.proto

```diff
@@ -141,6 +143,22 @@ message DatabaseInformation {
   bool collected_local_catalog_data = 11;
 }
 
+message DatabaseStatistic {
+  int32 database_idx = 1;
+
+  int32 frozenxid_age = 2; // The age of frozen xact ID for this database (relative to latest stable xid)
+  int32 minmxid_age = 3; // The age of minimum multixact ID for this database (relative to latest stable mxid)
+  int32 xact_commit = 4; // Number of transactions in this database that have been committed
+  int32 xact_rollback = 5; // Number of transactions in this database that have been rolled back
```

Comment:
Memo (for future me):
I kept all of them as `int32` since the age value could actually go negative ([postgres code also returns as int32](https://github.com/postgres/postgres/blob/4a29eabd1d91c5484426bc5836e0a7143b064f5a/src/backend/utils/adt/xid.c#L110). It could go negative with wraparound (e.g. current xid being 10 after wraparound and the frozenxid can be 3,999,999,999).
For `xact_commit` and `xact_rollback`, we also do subtraction on the collector side, and while this should never really go negative, it's still nice to be `int32`.

Also Lukas pointed out that using `uint32` here doesn't mean that we could actually take advantage of it while we persist these data in Postgres, as Postgres doesn't support unsigned integer type. It might be the best not to use unsigned integer in Snapshot to make our life simpler.